compiler/onecc-docker/onecc-docker

```diff
@@ -16,10 +16,43 @@
 
 import sys
 import subprocess
-import json
 import requests
 import os
 import argparse
+import re
+
+
+class RequestHandler:
+    def __init__(self, token=None, timeout=None):
+        if token:
+            self.headers = {"Authorization": "Bearer {}".format(token)}
+        else:
+            self.headers = {}
+        self.timeout = timeout or 5
+
+    def make_request(self, url):
+        try:
+            response = requests.get(url, headers=self.headers)
+            response.raise_for_status()
+            print("[onecc-docker] Request succeeded: {}".format(response.status_code))
```

Comment:
```suggestion
```

`onecc-tool` is an official tool for user. So we have to arrange the message well. For now, it prints the `[onecc-docker] Request succeeded: 200` message and it seems to be a log of the development process.

Reply:
Sure. let me remove it.